.github/workflows/build-docker-image.yml

```diff
@@ -20,6 +20,9 @@ jobs:
         username: ${{ secrets.DOCKERHUB_USERNAME }}
         password: ${{ secrets.DOCKERHUB_TOKEN }}
 
+    - name: Verify project
```

Comment:
I think this should be part of another workflow, right? ci.yml maybe? so, PRs are checked instead of doing it when the image is built and publish

Reply:
I had that thought too, but I checked this workflow is run on PRs, which is weird to build the image on them

Reply:
just saw the published 🤦🏽‍♂️

Reply:
I'd say that we should move the publish to a different pipeline that runs on tags, only

Reply:
Yes, sounds like the simpler approach 👍